__DEFINES/materials.dm

```diff
@@ -89,3 +89,4 @@
 #define CC_PER_SHEET_GINGERBREAD CC_PER_SHEET_METAL
 
 #define CC_PER_U 10 //How many cc per 1 u of reagent in eg. a glass of water or a human's bloodstream.
+#define SHEET_PER_U 20 //Same as above, but for making sheets
```

Comment:
should be U_PER_SHEET but that's a nitpick

Reply:
actually that is more correct, will implement asap. next time try using the suggested change feature though it saves people a lot.

Reply:
yeah you're right